velox/functions/lib/DateTimeFormatter.cpp

```diff
@@ -941,31 +1025,35 @@ std::string DateTimeFormatter::format(
   const date::year_month_day calDate(daysTimePoint);
   const date::weekday weekday(daysTimePoint);
 
-  std::string result;
+  char* resultBuffer = const_cast<char*>(result.data());
+  common::OutputByteStream outStream(resultBuffer);
```

Comment:
Can we use something from the standard library?

Reply:
Thanks for your comment! Not sure which standard library can be a replacement. And I just extended `OutputByteStream` to support appending multiple same char, required by outputting result inside `padContent()`. Maybe, we can keep the use of it.